### ISSUE TITLE
Exception in thread SpotifyEventLoop

### ISSUE THREAD

**Issue by OmgImAlexis** [jrnewell/spotify-ripper#93]
Running version 2.3.2 on OSX.

````
ID3 v2.4: 7 frames
Writing ID3 version v2.4
-------------------------------------------------------------------------------
Logging out...
Exception in thread SpotifyEventLoop (most likely raised during interpreter shutdown):
Traceback (most recent call last):
  File "/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 810, in __bootstrap_inner
````

**Comment by jrnewell** [comment]
Does this happen often for you or only once?

**Comment by jrnewell** [comment]
Also can you post that the Spotify URI that it was ripping when this occurred? Thanks!

**Comment by OmgImAlexis** [comment]
When I logged this it was happening with every 2nd - 3rd song. I was trying this url ``track:3dNjUFt6EFU4Gq6Q5vfJqf``.

Doesn't seem to be an issue anymore.

**Comment by blackleopard32** [comment]
Happend to me too.
Im running version 2.3.3 on Raspbian
The URI i was using: spotify:artist:6wWVKhxIU2cEi0K81v7HvP

**Comment by Drezzler** [comment]
This has been happening to me as well ..

**Comment by Victortjeuh** [comment]
# Bumping this, still an existent problem, please don't ignore us!

First of all, thank you, developer, for making this possible! I just installed the spotify ripper on a fresh Ubuntu install in VirtualBox (did updates and everything) and I followed the guide on installing spotify ripper on Ubuntu. This all went well, I downloaded a key file from the Spotify website and moved it into the ~/.spotify-ripper folder. Don't have a config.ini file.

This is what I get:
![Error](http://i.imgur.com/QWwPgHG.png)

I'm 100% sure that I entered the correct password (I've tried more than 5 times). I have pip and pyenv installed. Ubuntu version is 14.04 LTS x64. I tried to update python, spotify ripper. I tried to create a config.ini with all the possible options, but none of these had any affect.

What on earth is this problem?

**Comment by jrnewell** [comment]
Hey, do you still have this issue?  I was on vacation this weekend so I
didn't get to look into it yet.  Let me know....

**Comment by jrnewell** [comment]
I think this should be fixed in ``2.6.6`` and later.
